packages/addresses/src/base58.ts

```diff
@@ -52,3 +52,18 @@ export function getBase58EncodedAddressComparator(): (x: string, y: string) => n
         usage: 'sort',
     }).compare;
 }
+
+export async function createBase58AddressWithSeed(
```

Comment:
For pedantry:

```suggestion
export async function createBase58EncodedAddressWithSeed(
```

Reply:
Akshually; I just renamed everything to `*Address*` didn't I? So `createAddressWithSeed` is the thing that would match everything else now.